.github/CONTRIBUTING.md

```diff
@@ -1,2 +1,59 @@
-*	**PR:** Test your code, describe your change, request it.
-*	**Issue:** It should be descriptive enough.
+# Contributing 
+Our commit history is a great example of why some structure when contributing is necessary.
+This document describes how you should go about contributing to the repo, please read all relevant parts before contributing.
+
+## Feature branches
+As both master and development are protected, any changes will have to be made through a feature branch. A feature branch should have the following characteristics through its lifetime:
+
+- The branch is linked to at least one issue.
+- It should only have one clear and achievable goal.  
+eg. "Add cinfo command" and not "Change license and fix the call command"
+- A feature branch should be named after the issue like: `<nr>-<lowercase-name>`.  
+eg. you have an issue nr 301 named "Add feature branches to contributing", then your branch should be named: `301-add-feature-branches-to-contributing`.
+
+## Issues
+### Creation
+- Make sure there is not already an issue regarding your topic.
+- Keep the title short, descriptive and [imperative].
+- Answer the following questions in the description:
+  - What is the current situation?
+  - Why should this be changed?
+  - What changes are necessary?
+- Add any relevant tags.
+- If you are planning to resolve the issue yourself, at least partly, then assign yourself to the issue.
+
+### Commenting
+- All comments should be written in English.
+- Be sure to follow our [Code of Conduct].
+- Keep any discussion on-topic, or move it to another issue/platform.
+
+## Pull requests
+
+### Creation
+- Keep the title short, descriptive and [imperative].
+- Write a short description of your changes.  
+If your PR solves any issue(s), make sure to add `resolves #<issue nr>` at the end.
+- Add any relevant tags.
+
+### Review
+These steps have been written to go from quick and easy to more rigorous.  
+If at any stage you find an issue, you can choose to stop at that stage until a fix has been pushed. Do make sure to go through the entire process of the stage you found an issue at. 
```

Comment:
Maybe add something like "If you need any help testing your changes, let us know and a maintainer/contributor will help"?

Reply:
Only org members/collaborators can review anyway. That section is really only for those people.
I did think of adding something like "when you are ready, request review", but they wouldn't know who to ask. You just have to add the 'finished' tag when you want review.

Reply:
Alright I just checked and it seems like you can request a team to review, I'll put us in a team so they can request all of us at once.

Reply:
I could be wrong, but I don't think external contributors can add tags?

Reply:
Oh that could be true also